events/apigw.go

```diff
@@ -80,7 +80,9 @@ type APIGatewayV2HTTPRequestContext struct {
 
 // APIGatewayV2HTTPRequestContextAuthorizerDescription contains authorizer information for the request context.
 type APIGatewayV2HTTPRequestContextAuthorizerDescription struct {
-	JWT APIGatewayV2HTTPRequestContextAuthorizerJWTDescription `json:"jwt"`
+	JWT    *APIGatewayV2HTTPRequestContextAuthorizerJWTDescription `json:"jwt,omitempty"`
+	Lambda map[string]interface{}                                  `json:"lambda,omitempty"`
```

Comment:
Lambda authorizers can return plain strings... or ints, bools, arrays - right?

Would `interface{}` be more correct here?

Reply:
Authorizer always returns k/v pair, where key will always be a `string`.